_articles/zh-hant/best-practices.md

```diff
@@ -265,6 +265,6 @@ fork一個專案不什麼壞事情。能複製並且修改別人的程式是開
 
 休假不僅適用於度假。如果你週末不想做開源專案的工作了，或者在本該工作的時候不想幹活了，和別人說，這樣他們知道什麼時候不該打擾你。
 
-## 首先照顧好自己！
+## 不必事必躬親!
```

Comment:
Autotranslation says the existing phrase means "First take care of yourself!" while this PR changes the meaning to "You don't have to do everything!"

Compare with https://opensource.guide/best-practices/#take-care-of-yourself-first -- the existing translation seems to be closer to that.

Perhaps a contributor to the zh-hant translation @felixshai @jyangchisyan can comment on whether this PR's edit is an improvement, as explained by @KaynenStrife in the PR comment "I propose changing the words to the current version which is more compatible with the zh-hans version because it conveys a much more formal intent."

The zh-hans translation https://opensource.guide/zh-hans/best-practices/#%E4%B8%8D%E5%BF%85%E4%BA%8B%E5%BF%85%E8%BA%AC%E4%BA%B2 does autotranslate (probably imperfectly, blame to autotranslation) to something closer to this PR -- "Don't have to do everything"